### DISCUSSION TITLE
get current user name and user id in new version

### DISCUSSION THREAD

**Discussion by abualmazen** [serenity-is/Serenity#6805] [General]
how i can get current user name and user id in new version
in previous version  by this code 
var user = Context.User.GetIdentifier();
                var user2 = Context.User.Identity.Name;
but not work in new version

**Comment by ShifanilaSenthil** [comment]
Have you tried removing the context? 
Var user1=User.Getidentifier()

**Comment by abualmazen** [comment]
thank you it is work fine
